code/DepMapGeneEss/ConvertGeneEssInputData.R

```diff
@@ -0,0 +1,70 @@
+library(tidyverse)
+
+dataFolder = "C:/Work/MatlabCode/components/human-GEM/Human-GEMDepMapEval/Human-GEM/code/DepMapGeneEss/data/"
```

Comment:
I'm guessing this was meant to be a relative folder.

Reply:
The problem with R is that there is no way to solve that as far as I know? Please enlighten me if you know.

Reply:
I've had to add a separate suggestion, so I'll close this one.